src/appleseed.cli/commandlinehandler.cpp

```diff
@@ -104,9 +104,9 @@ CommandLineHandler::CommandLineHandler()
         &m_samples
             .add_name("--samples")
             .add_name("-s")
-            .set_description("set the minimum and maximum numbers of samples per pixel")
-            .set_syntax("min max")
-            .set_exact_value_count(2));
+            .set_description("set the number of samples per pixel")
+            .set_syntax("n")
+            .set_exact_value_count(1));
```

Comment:
I'm not sure this will work. Check out `apply_rendering_settings_command_line_options` in `main.cpp`.

Reply:
This is ok but you need to update places where `m_samples` is used.

Reply:
I didn't see any conflicts with m_samples, but I'll take a look again. Is there anywhere that immediately seemed like a change was necessary?

Reply:
Not sure but the line `g_cl.m_samples.values()[1]` won't work anymore i suppose ? We now expect only one value and you access the second value with `[1]`.

Reply:
Yes, for sure the code that uses this option needs to be updated.

Reply:
True g_cl.m_samples.values()[1] would work, but it would return the 2nd digit in the integer which is not what we want. I'll go through and see what needs to change.

Reply:
It looks like `src/appleseed.cli/main.cpp` was the only user of `m_samples`. Other occurrences of `m_samples` seem to be a different type. Please correct me if I'm wrong.